src/components/thumbnail/documents-type-collection.tsx

```diff
@@ -43,13 +43,13 @@ function getNewDocumentLabel(section: NavTabSectionModelType , appConfigStore: A
 
 function getSectionDocs(section: NavTabSectionModelType, documents: DocumentsModelType, user: UserModelType,
   isTeacherDocument: (document: DocumentModelType) => boolean) {
-  const publishedDocs: { [source: string]: DocumentModelType } = {};
   let sectDocs: DocumentModelType[] = [];
   (section.documentTypes || []).forEach(type => {
     if (isUnpublishedType(type)) {
       sectDocs.push(...documents.byTypeForUser(type as any, user.id));
     }
     else if (isPublishedType(type)) {
+      const publishedDocs: { [source: string]: DocumentModelType } = {};
```

Comment:
As previously implemented, `publishedDocs` was defined outside the types loop in line 47. This meant that since the documents from previous types were never removed, when `publishedDocs` were added to `sectDocs` in line 69, the same documents could be added multiple times. Moving `publishedDocs` inside the loop prevents this from occurring.